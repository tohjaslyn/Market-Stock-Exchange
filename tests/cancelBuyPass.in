add trader1 100
buy trader1 ABC 10 1.0
cancel buy 0000
exit
