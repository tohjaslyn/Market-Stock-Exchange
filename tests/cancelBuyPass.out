$ Success.
$ No trades could be made, order added to buy book.
$ Order successfully cancelled.
$ Have a nice day.
